package.json

```diff
@@ -12,14 +12,13 @@
     "build:sdk": "firebolt-openrpc --task sdk --source ./src/ --template ./src/template/js --output ./build/sdk/javascript/src --static-modules Platform --shared-schemas node_modules/@firebolt-js/schemas/src/schemas/",
     "build:docs": "firebolt-openrpc --task docs --source ./src/ --template ./src/template/markdown --output ./build/docs/markdown --shared-schemas node_modules/@firebolt-js/schemas/src/schemas/ --as-path",
     "build:wiki": "firebolt-openrpc --task docs --source ./src/ --template ./src/template/markdown --output ./build/docs/markdown --shared-schemas node_modules/@firebolt-js/schemas/src/schemas/",
-    "dist:sdk": "npm run validate && npm run build:sdk && npm run pack && npm run test",
-    "dist:d": "npm run validate && firebolt-openrpc --task declarations --source ./src/ --output ./dist/firebolt.d.ts --shared-schemas node_modules/@firebolt-js/schemas/src/schemas/",
+    "dist:sdk": "npm run validate && npm run build:sdk && cp -R ./build/sdk/javascript/src ./dist/lib && npm run test",
```

Comment:
One minor concern with using linux shell commands here: Windows users will have to use git bash or similar. Most devs do anyway, but one of the developers hit this issue with another repo. (We might want to add a sentence about running in a shell that supports linux commands.)